sync3/conn.go

```diff
@@ -78,11 +78,15 @@ func (c *Conn) OnUpdate(ctx context.Context, update caches.Update) {
 }
 
 func (c *Conn) tryRequest(ctx context.Context, req *Request) (res *Response, err error) {
+	// TODO: include useful information from the request in the sentry hub/context
 	defer func() {
 		panicErr := recover()
 		if panicErr != nil {
 			err = fmt.Errorf("panic: %s", panicErr)
 			logger.Error().Msg(string(debug.Stack()))
+			internal.GetSentryHubFromContextOrDefault(ctx).RecoverWithContext(ctx, panicErr)
```

Comment:
This is a bit of a mouthful. Not wedded to the name.

I only need the fallback hub for one test case in this PR. However in an upcoming PR I want to send events to sentry with a context that might have a hub or might not.

Reply:
Agreed!